### PR TITLE
CCIP-3799 add condition for Execution plugin in _afterOCR3ConfigSet

### PR DESCRIPTION
**Pull request by 0xsuryansh**
This PR fixes a missing condition for the Execution plugin in the _afterOCR3ConfigSet function. Now, the function correctly reverts if signature verification is enabled for the Execution plugin. The updated logic ensures that:

Commit Plugin: Signature verification must be enabled.
Execution Plugin: Signature verification must be disabled.

### CODE REVIEW

File: contracts/src/v0.8/ccip/offRamp/OffRamp.sol [hunk 1]
```diff
@@ -63,7 +63,8 @@ contract OffRamp is ITypeAndVersion, MultiOCR3Base {
   error ZeroAddressNotAllowed();
   error InvalidMessageDestChainSelector(uint64 messageDestChainSelector);
   error SourceChainSelectorMismatch(uint64 reportSourceChainSelector, uint64 messageSourceChainSelector);
-  error SignatureVerificationDisabled();
+  error SignatureVerificationRequired(uint8 pluginType);
```
Comment (RensR) [inline contracts/src/v0.8/ccip/offRamp/OffRamp.sol]: let's remove the args as these can only be thrown with a single arg which we can make a part of the error name

File: contracts/src/v0.8/ccip/offRamp/OffRamp.sol [hunk 2]
```diff
@@ -882,16 +883,23 @@ contract OffRamp is ITypeAndVersion, MultiOCR3Base {
   function _afterOCR3ConfigSet(
     uint8 ocrPluginType
   ) internal override {
+    bool isEnabled = s_ocrConfigs[ocrPluginType].configInfo.isSignatureVerificationEnabled;
```
Comment (RensR) [inline contracts/src/v0.8/ccip/offRamp/OffRamp.sol]: can you rename this to sigVerificationEnabled or something similar?

File: contracts/src/v0.8/ccip/offRamp/OffRamp.sol (hunk 1 above)
Comment (0xsuryansh) [reply contracts/src/v0.8/ccip/offRamp/OffRamp.sol]: https://github.com/smartcontractkit/chainlink/pull/14951/files#diff-865ecf8626037bea80482f856f6aff9bdd5a948982897dab6e2b4ff335af032aR66-R67

File: contracts/src/v0.8/ccip/offRamp/OffRamp.sol (hunk 2 above)
Comment (0xsuryansh) [reply contracts/src/v0.8/ccip/offRamp/OffRamp.sol]: https://github.com/smartcontractkit/chainlink/pull/14951/files#diff-865ecf8626037bea80482f856f6aff9bdd5a948982897dab6e2b4ff335af032aR886

**Review by RensR — APPROVED** [review]

**Review by RayXpub — APPROVED** [review]

**Pull request merged**